### DISCUSSION TITLE
PageTransitionType.fade Not working

### DISCUSSION THREAD

**Discussion by vpdevportal** [nylo-core/nylo#158] [Q&A]
In the latest versions PageTransitionType.fade is not working

**Comment by agordn52** [comment]
Hi @vpdevportal,

Is this for iOS and Android?

Also, Nylo uses [page_transition](https://pub.dev/packages/page_transition) for transitions.
The author of that package hasn't updated it in a while so we may end up building our own custom transitions.

**Reply by vpdevportal** [reply]
Mine is a cross-platform app used for iOS and the Web. Right now I am facing this issue in iOS.

**Reply by agordn52** [reply]
Ahh I see,

I'll try to work on a custom transition library for Nylo to avoid [page_transition](https://pub.dev/packages/page_transition) since I don't think the maintainer will update it.

**Comment by dylandamsma** [comment]
Second this btw :)

**Reply by agordn52** [reply]
v6 should hopefully include the new custom library for page transitions. If anyone can send a PR to fix this, it'd be appreciated. The current package we're using for transitions is [page_transition](https://pub.dev/packages/page_transition). It doesn't seem to be maintained anymore by the developer.
